documentation/backup.md

```diff
@@ -0,0 +1,49 @@
+# Backup Puppet Enterprise using the PEADM module
+
+## What is being backed up?
+
+By default the `peadm::backup` plan will backup the following items:
+
+1. Orchestrator
+2. PuppetDB
+3. RBAC
+   - Also, It copies the LDAP secret key if it present
+4. Activity
+5. Classification
+
+Optionally you can also backup:
+
+1. CA (CA and SSL certificates)
+
+**Note:** It is important to note that the `peadm::backup` plan in its current version is not as granular as you can get when you manually run [the `puppet-backup create` command.](https://puppet.com/docs/pe/2021.7/backing_up_and_restoring_pe.html#back_up_pe_infrastructure)
+
```

Comment:
Where do we explain what the added value is of `peadm::backup` as opposed to `puppet-backup create` method?

Reply:
Good point I think if we highlight the focus of the original puppet-backup is to do like for like in place restores while the intention of this plan is to backup only the user data and not infrastructure data making it able to restore it to another set of Puppet infrastructure regardless of location and type. HIghlighting its compatibility with XL too

Reply:
Can we describe this more specifically? What data does `peadm::backup` leave out which `puppet-backup` includes? Can we also add an explicit scenario on how to go about restoring a backup onto a new environment?

Reply:
The core idea of the PEADM backup script is a focus on being able to separate the user/customer data from the infrastructure data. This contrasts with the PE backup script which is more focused on a traditional backup and restore after a system failure where will be restoring the same infrastructure. It therefore targets things like backing up postgress rather than the actual data you want.

This can be seen when restoring node groups with PEADM backup only restores the non-PEinfrastructure source groups and restores them to target. 

PE backup is not capable of restoring to a DR setup and requires you to provision a new DR setup whereas PEADM restore will reinitialise replicas as part of the process

One of the other key differences with the peadm backup is it is compatible with the xlarge configurations as it can communicate with the remote postgres server using the temporary escualted privlidges

I wouldn't actually say it was less granular than the scope of PEADM backup is more data focused like classification rather than object storage focused as PE backup is with scopes like database

Reply:
@davidsandilands @timidri I used David's core idea for the PEADM backup plan.

https://github.com/puppetlabs/puppetlabs-peadm/blob/02646f5df70a732ad236a97ee5cf1284018818f4/documentation/backup.md

Let me know what you think! :)

Reply:
@mcka1n that looks good to me!